README.md

```diff
@@ -49,363 +35,59 @@ Yomitan provides advanced features not available in other browser-based dictiona
 [![Dictionary options](img/ss-dictionaries-thumb.png)](img/ss-dictionaries.png)
 [![Anki options](img/ss-anki-thumb.png)](img/ss-anki.png)
 
-## Table of Contents<!-- omit from toc -->
-
-- [Project Introduction](#project-introduction)
-- [Tool Introduction](#tool-introduction)
-- [Installation](#installation)
-- [Migrating from Yomichan](#migrating-from-yomichan)
-  - [Exporting Data](#exporting-data)
-  - [Custom Templates](#custom-templates)
-- [Dictionaries](#dictionaries)
-- [Basic Usage](#basic-usage)
-  - [Importing Dictionaries](#importing-dictionaries)
-  - [Importing and Exporting Personal Configuration](#importing-and-exporting-personal-configuration)
-- [Custom Dictionaries](#custom-dictionaries)
-- [Anki Integration](#anki-integration)
-  - [Flashcard Configuration](#flashcard-configuration)
-  - [Flashcard Creation](#flashcard-creation)
-- [Keyboard Shortcuts](#keyboard-shortcuts)
-- [Advanced Options](#advanced-options)
-  - [Parse sentences using MeCab](#parse-sentences-using-mecab)
-- [Frequently Asked Questions](#frequently-asked-questions)
-- [Licenses](#licenses)
-- [Third-Party Libraries](#third-party-libraries)
+## Documentation<!-- omit from toc -->
+
+- [Migrating from Yomichan](./docs/yomichan-migration.md#migrating-from-yomichan)
```

Comment:
These links only appear once and the user is likely to miss them. Could probably keep a concise summarized section for each of them, so it flows better (for example, anki integration is a core feature of yomitan but very unnoticeable)

Reply:
lgtm